src/Chapter1.hs

```diff
@@ -568,7 +564,14 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool 
+isVowel c 
+  | c == 'a' = True 
+  | c == 'e' = True 
+  | c == 'i' = True 
+  | c == 'o' = True 
+  | c == 'u' = True 
+  | otherwise = False 
```

Comment:
Nice one! 👍🏼 
As `==` returns `Bool` itself, you can just return the result of this function:

```haskell
isVowel c = c == 'a' || c == 'e' || ...
```